ApacheConnector/CMakeLists.txt

```diff
@@ -19,7 +21,14 @@ target_include_directories(mod_poco
 		${APACHE2_INCLUDE_DIRS}
 		${CMAKE_CURRENT_SOURCE_DIR}/src
 )
-target_link_libraries(mod_poco PUBLIC Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
+set(DEPLIBS Poco::Util Poco::Net Apache::Apr Apache::Aprutil)
+if(WIN32)
+	list(APPEND DEPLIBS Apache::httpd)
+endif(WIN32)
+target_link_libraries(mod_poco PUBLIC ${DEPLIBS})
+
+file(MAKE_DIRECTORY "include/Poco")
```

Comment:
Not sure why we need this ? Should be created with this `POCO_INSTALL` macro

Reply:
@felfert what is the purpose of this line?